integration/spark/src/main/scala/org/apache/spark/util/AlterTableUtil.scala

```diff
@@ -442,7 +442,11 @@ object AlterTableUtil {
         // update schema for long string columns
         updateSchemaForLongStringColumns(thriftTable, longStringColumns.get)
       } else if (propKeys.exists(_.equalsIgnoreCase("long_string_columns") && !set)) {
-        updateSchemaForLongStringColumns(thriftTable, "")
+        val varcharColumns = carbonTable.getTableInfo.getFactTable.getListOfColumns.asScala.toList
```

Comment:
why not directly check the table properties to see long_string property exist before or not ? If you have lot of columns, this will be slow compared to table property look up.

Reply:
Done